templates/partials/scripts-footer.hbs

```diff
@@ -1,6 +1,5 @@
-<script src="https://unpkg.com/scrollreveal/dist/scrollreveal.min.js" defur></script>
+<script src="https://unpkg.com/scrollreveal/dist/scrollreveal.min.js" defer></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.20.4/TweenLite.min.js" defer></script>
```

Comment:
so we currently have the actual copy of the lib _and_ the lib from CDN.

I'm really keen as part of this PR to _not_ use the CDN (since it will likely load after our scripts and fuck everything up - _maybe_), and try to work out a solution to get the npm package, copy it over via GULP task so we can upgrade in the same way we upgrade all the rest of the packages.

I hope this makes sense.

Reply:
As discussed we need to give the bundling a rethink and will done in another story